src/config/config_module.rs

```diff
@@ -153,3 +163,93 @@ impl From<Config> for ConfigModule {
         ConfigModule { config, input_types, output_types, ..Default::default() }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    mod extensions {
+        mod merge_right {
```

Comment:
Do we need these tests for merge_right? We are using macros now, so we should be able to depend on them.

Reply:
I've added impl MergeRight for external type FileDescriptorSet